app/boards/shields/tester_pro_micro/tester_pro_micro-layouts.dtsi

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2025 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include <physical_layouts.dtsi>
+
+/ {
+    tester_postition_map {
```

Comment:
```suggestion
    tester_position_map {
```

Reply:
I found the same typo in the XIAO file (but not in the same spot, so not a victim of copy-paste, but a victim of my own clumsy fingers). Should be clean now!